.gitignore

```diff
@@ -2,24 +2,28 @@
 /source/config.inc.php
 
 # Log
-/source/log/
+!/source/log/
+/source/log/*
 !/source/log/.htaccess
 
 # Cache
-/source/cache/
+!/source/cache/
+/source/cache/*
 !/source/cache/.htaccess
 !/source/cache/dir.txt
 
 # Export
-/source/export/
+!/source/export/
+/source/export/*
 !/source/export/.gitkeep
 
 # Temporary files
-/source/tmp/
+!/source/tmp/
+/source/tmp/*
 !/source/tmp/.htaccess
 
 # Composer
-/vendor/
+#/vendor/
```

Comment:
Just out of curiosity: Why would you not ignore the `vendor` directory? As not only it's contents, but the directors itself is fully managed by composer.

Reply:
oh yeah sry, that's a mistake I added the vendor directory to my git repo.... so you can decomment this so that it is active
^^